proposals/0000-swiftpm-script-support.md

```diff
@@ -38,8 +38,10 @@ The `@package(...)`syntax will allow the same parameters as SwiftPM's [Package D
 
 #### Basic usage
 The most common case is expected to be importing a dependency where the product name is equivalent to the imported module name.
+
+Specifying the version of the package is optional. If omitted, SwiftPM will resolve the latest version of the dependency.
 ```swift
-@package(url: "https://github.com/jpsim/Yams.git", from: "2.0.0")
```

Comment:
We should put non-optional version in the alternative considered and explain why we made that decision.